java/core/src/main/scala/com/azavea/pdal/Main.scala

```diff
@@ -0,0 +1,54 @@
+package com.azavea.pdal
```

Comment:
We should change the namespacing from this to hobu (since the project is not azaveas).

@hobu we need to namespace, so should we do

```
package com.hobu.pdal
```

or something else?